### PR TITLE
Enable v2 endpoint and handle minSupportedVersion

### PR DESCRIPTION
**Pull request by SlayterDev**
https://app.asana.com/0/1200890834746050/1201683113648489/f

This PR pushes config to v1 and v2 directories. If a feature contains the `minSupportedVersion` key we'll delete this key from v1 and disable the feature.

### CODE REVIEW

File: index.js
Comment (jonathanKingston) [inline index.js:75]: Can we make the `existsSync` and `mkdirSync` a function and call it for all three directories? Locally this broke for me as I already had a generated/ but not the other dirs. Ideally users of the config shouldn't need to understand about the generation process.


Also worth noting this path is mentioned in the extensions CONTRIBUTING.md for local development.  (happy to add this to the task of integrating the update to v2)
Perhaps other projects are depending on the generated directory being the route?

File: tests/schemas/feature.json
Comment (jonathanKingston) [inline tests/schemas/feature.json:12]: 👍🏻

File: index.js
Comment (SlayterDev) [reply index.js:75]: I'll get those into a function.

Regarding the change of local path yeah that's something we should probably delegate to v2 integration projects.

**Review by jonathanKingston — APPROVED** [review]
LGTM if you're merging it ensure the output of the generated build matches before and after merge please :)

**Pull request merged**
